content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md

```diff
@@ -61,6 +61,24 @@ If you create a new clone of the repository, you won't lose any of your Git hist
 
    The repository should now only contain the files that were in your subfolder(s).
 
+   If you want one specific subfolder to be the new root folder of the new repository, the following command can be used:
+
+   ```shell
+   $ git filter-repo --subdirectory-filter FOLDER-NAME
+   # Filter the specific branch by using a single sub-directory as the root for the new repository
+   > Parsed 82 commits
+   > New history written in 0.44 seconds; now repacking/cleaning...
+   > Repacking your repo and cleaning out old unneeded objects
+   > HEAD is now at 968d5a5 +: Print example / Notifier part 1
+   > Enumerating objects: 101, done.
+   > Counting objects: 100% (101/101), done.
+   > Delta compression using up to 6 threads
+   > Compressing objects: 100% (58/58), done.
+   > Writing objects: 100% (101/101), done.
+   > Total 101 (delta 44), reused 73 (delta 36), pack-reused 0
+   > Completely finished after 1.01 seconds.
```

Comment:
I realise you're following the convention the article already has here from lines 58 and 59, but I'd recommend that we delete these and _also_ lines 58 and 59, as the output of this command may change at some point, and the commits in this example don't necessarily match lines 58/59.

```suggestion
```